app/Commands/Traits/ArtisanCall.php

```diff
@@ -19,10 +20,23 @@ public function artisanCall($command, array $arguments = [])
 
     public function artisanCallNoOutput($command, array $arguments = [])
```

Comment:
This was broken, `fwd start` wasn't working. I had to make this ugly workaround because sometimes the artisan call runs using `Process` and sometimes `CommandExecutor`, so `fwd reset` was printing output for `yarn install` for example.

Reply:
we might want to add a deprecation note for Process, and target to removing it soon at 1.1 - to simplify this. Maybe even sooner (now? - custom commands will become outdated and break... better now than later?)

Reply:
I agree to deprecate and remove Process class asap, avoid issues and simplifying maintenance...